crawl-ref/source/describe.cc

```diff
@@ -1699,6 +1700,39 @@ static string _describe_armour(const item_def &item, bool verbose)
                 description += "       Evasion: "
                             + to_string(evp / 30);
             }
+
+            if (is_body_armour)
+            {
+                const armour_type body_type =
+                    static_cast<armour_type>(item.sub_type);
+                const int item_gdr_base = armour_gdr(body_type, false);
+                const int item_gdr_actual = armour_gdr(body_type, true);
+                const bool worn = _you_are_wearing_item(item);
+                if (item_gdr_actual) {
```

Comment:
The curly brace here is on the same line but they are on the next line elsewhere.